documentation/source/simulator_support.rst

```diff
@@ -121,6 +121,16 @@ To get waveforms in VCD format, add Verilator's trace option(s) to the
 To set the same options on the command line, use ``EXTRA_ARGS="--trace --trace-structs" make ...``.
 A VCD file named ``dump.vcd`` will be generated in the current directory.
 
+If you wish to use FST-Tracing (much smaller file size), which is supported by GTKWAVE, then use
+
+  .. code-block:: make
+
+    EXTRA_ARGS += --trace-fst --trace-structs -CFLAGS -DVM_TRACE_FST
```

Comment:
Since https://github.com/verilator/verilator/pull/2592 will it work like this?
```suggestion
    EXTRA_ARGS += --trace-fst --trace-structs
```

Reply:
see below.

Yes it does, but I would not move to that just yet, we need to mention you can 'workaround' a slightly older verilator version like this.